src/components/shared/ObjectInspector.js

```diff
@@ -138,7 +138,9 @@ class ObjectInspector extends Component {
           focused,
           "default-property": isDefault(item)
         }),
-        style: { marginLeft: depth * 15 },
+        style: {
+          marginLeft: nodeIsPrimitive(item) ? depth * 15 + 15 : depth * 15
```

Comment:
perhaps: `depth * 15 + (nodeIsPrimitive(item) ?  15 : 0)` to be a little cleaner